### PR TITLE
Changed nGraph code style to Google.

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Used clang-format config from IE samples
 - Fixed code style check for tests
 - Applied code style

### CODE REVIEW

**Comment by tsocha** [comment]
Clang format failed, is the style file updated?

**Comment by ilyachur** [comment]
> 
> 
> Clang format failed, is the style file updated in the CI?

I just fixed code style with disabled python.

**Comment by ilya-lavrenov** [comment]
First of all we need to adjust current scripts to match IE code style. After this - apply common code style to ngraph

**Comment by ilyachur** [comment]
> First of all we need to adjust current scripts to match IE code style. After this - apply common code style to ngraph


I took this script from our samples, in samples we are using the script which was aligned some time ago with IE style (I did such experiments around a year ago).
But I have removed the option:
```
SpaceBeforeCpp11BracedList: true
```
Because it causes some issues with nGraph tests.

**Comment by ilya-lavrenov** [comment]
IE != IE samples. 
IE uses cpplint. So, we need to align clang-config with cpplint-config and run clang on IE first to make sure no significant changes. Only after that this can be applied to the whole OpenVINO

**Comment by ilyachur** [comment]
> IE != IE samples.
> IE uses cpplint. So, we need to align clang-config with cpplint-config and run clang on IE first to make sure no significant changes. Only after that this can be applied to the whole OpenVINO

Enabled clang-format for IE public API, IE plugin API and IE core in this PR: https://github.com/openvinotoolkit/openvino/pull/6938

File: ngraph/.clang-format [hunk 1]
```diff
@@ -1,54 +1,31 @@
...
+AllowShortLambdasOnASingleLine: Empty
+AllowShortLoopsOnASingleLine: false
 
-BinPackArguments: false
-BinPackParameters: false
+AlwaysBreakBeforeMultilineStrings: false
 
-BreakBeforeBraces: Allman
-BreakConstructorInitializersBeforeComma: true
+ColumnLimit: 160
```
Comment (tomdol) [inline ngraph/.clang-format:21]: Can we go down to 120 here? A single file with so many columns is difficult to display on FullHD screen without horizontal scrolling(in an IDE with project tree open).

Comment (ilyachur) [reply ngraph/.clang-format:21]: Such limit is using in the IE. I propose to be aligned between components.

Comment (tsocha) [reply ngraph/.clang-format:21]: So maybe other components could be changed as well.

Comment (ilyachur) [reply ngraph/.clang-format:21]: I will update the limit to 120 symbols

Comment (ilyachur) [reply ngraph/.clang-format]: The config was updated.

**Comment by ilyachur** [comment]
@tomdol @postrational @mbencer @blesniewski @tsocha @jiwaszki @sdurawa @t-jankowski @bszmelcz @mitruska Please have a look

**Review by tomdol — APPROVED** [review]

**Review by mbencer — APPROVED** [review]

File: ngraph/core/include/ngraph/descriptor/input.hpp [hunk 2]
```diff
@@ -10,104 +10,119 @@
...
+        return *m_output;
+    }
+    /// \return the connected output
+    Output& get_output() {
+        return *m_output;
+    }
+    /// \return true if an output is connected to the input.
+    bool has_output() const {
+        return m_output != nullptr;
+    }
```
Comment (mitruska) [inline ngraph/core/include/ngraph/descriptor/input.hpp:56]: I can just share my opinion that a header file with one-line functions allowed looks quite better for me, but I understand this is a necessary change;)

**Review by mitruska — APPROVED** [review]

**Review by jiwaszki — APPROVED** [review]

**Pull request merged**